xonsh/environ.py

```diff
@@ -1231,6 +1203,8 @@ def partial_format_prompt(template=DEFAULT_PROMPT, formatter_dict=None):
     except:
         return _failover_template_format(template)
 
+format_prompt = partial_format_prompt
```

Comment:
I guess I just wasn't entirely sure that dropping it would be harmless. It's still used - https://github.com/xonsh/xonsh/blob/c345694b/xonsh/base_shell.py#L239  and BTW seems like format specs do work for that case.